src/isar_turtlebot/turtlebot/turtlebot.py

```diff
@@ -46,6 +61,7 @@ def publish_task(self, task: Task) -> None:
                 target=task.target, current_pose=self.get_robot_pose()
             )
 
+        self.task_handler = self.task_handlers[type(task).__name__]
```

Comment:
Could this line be removed, as the task handler instance is already set at line 55?